ftool/__init__.py

```diff
@@ -59,15 +59,14 @@ def __init__(self, files, observable="SUEP_nconst_Cluster ", name = "QCD",
           for fn in self._files:
                _proc = os.path.basename(fn).replace(".root","")
                _file = uproot.open(fn)
-
                if not _file:
                     raise ValueError("%s is not a valid rootfile" % self.name)
 
                histograms = None
 
                _scale = 1
-               if ptype.lower() != "data":
-                    _scale = self.lumi* 1000.0 #
+               if self.name != "data":
```

Comment:
This should be kept as before.

Reply:
Thank you for spotting it, fixed in latest commit.